Vagrantfile

```diff
@@ -20,17 +20,39 @@ Vagrant.configure(2) do |config|
   # `vagrant box outdated`. This is not recommended.
   # config.vm.box_check_update = false
 
+
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine. In the example below,
   # accessing "localhost:8080" will access port 80 on the guest machine.
+
+  tomcatPort = ENV['TOMCAT_PORT']
+  if tomcatPort.nil?
+    tomcatPort = '8888'
+  end
+
+  transPort = ENV['NODEJS_PORT']
+  if transPort.nil?
+    transPort = '8094'
+  end
+
+  mergePort = ENV['P2P_PORT']
+  if mergePort.nil?
+    mergePort = '8096'
+  end
+
+  mapnikPort = ENV['NODE_MAPNIK_SERVER_PORT']
+  if mapnikPort.nil?
+    mapnikPort = '8000'
+  end
```

Comment:
This is not meant to be the place to specify ports. 

I threw these lines in to be able to spawn multiple vagrant jobs on the one machine. On my desktop, if I try to run another copy of vagrant hoot (from another directory) I get these errors:
```
==> default: Setting the name of the VM: hoot5_default_1460731267321_73061
Vagrant cannot forward the specified ports on this VM, since they
would collide with some other application that is already listening
on these ports. The forwarded port to 8888 is already in use
on the host machine.

To fix this, modify your current project's Vagrantfile to use another
port. Example, where '1234' would be replaced by a unique host port:
```

Thinking about it, I could put "custom" port options into a VagrantfileLocal which is probably a cleaner way to do this.